crates/rip_bin/src/main.rs

```diff
@@ -97,18 +96,19 @@ struct SDistResolution {
     only_sdists: bool,
 }
 
-impl From<SDistResolution> for resolve::SDistResolution {
```

Comment:
wdyt about renaming the SDistResolution from src/main.rs? into something like: DistResolution? or maybe ResolutionArgs? so we will have a distinction between cli args and actually library option and will not hide the name

Reply:
Yeah great idea! Let me change it!

Reply:
Done!